presto-main/src/test/java/io/prestosql/sql/planner/TestDomainTranslator.java

```diff
@@ -1582,6 +1582,39 @@ public void testLikePredicate()
         assertUnsupportedPredicate(not(like(C_VARCHAR, stringLiteral("abc\\_def"))));
     }
 
+    @Test
+    public void testStartsWithFunction()
```

Comment:
Could you add a test for generic function call (that it's not supported)?

Reply:
Added `testUnsupportedFunctions()` which tests some unsupported functions.